rust_team_data/src/v1.rs

```diff
@@ -175,7 +175,7 @@ pub struct Repo {
     pub members: Vec<RepoMember>,
     pub branch_protections: Vec<BranchProtection>,
     pub crates: Vec<Crate>,
-    pub environments: Vec<Environment>,
+    pub environments: IndexMap<String, Environment>,
```

Comment:
This is a breaking change. I think that we can leave the environments as they were in the v1 API, and just add a `branches` field to the environment.

Reply:
To clarify, the contents of the v1 API is used by several other projects. If this PR is merged as-is, those projects will try to deserialize this field as a `Vec` (because they will still have the old `serde` definition), but the API will now expose it as a JSON dictionary. So the repository endpoint will stop working for them.

Reply:
I'm not sure. Is there any way to confirm this? As environment feature was implemented very recently https://github.com/rust-lang/team/pull/2127, if we still have time, I would like to use the current structure because IMO this is how the schema should be.

I would love to hear your thoughts @Kobzol

Reply:
Hmm, in theory if the projects haven't updated their `team` version *yet*, it would be doable. Here is a list of some projects that use the API: https://github.com/search?type=code&q=%22rust_team_data+%3D+%22+org%3Arust-lang We'd need to check what version of `team` they use, if they use the repos endpoint.